src/flick/search/views.py

```diff
@@ -77,6 +77,7 @@ def get_tags_by_name(self, query):
     def get(self, request, *args, **kwargs):
         self.request = request
         query = request.query_params.get("query")
+        page = request.query_params.get("page", 1)
```

Comment:
so if we don't pass in a page, it'll default to 1?

Reply:
yup, the first page :)))